### ISSUE TITLE
[Feature Request] Automatically generated playlist covers

### ISSUE THREAD

**Issue by Currrupted** [zyrouge/symphony#310]
### Description

It was previously proposed (see #247 ) to implement the option to add a photo or cover to one's playlists. It would be great to allow Symphony to do that for you, by generating a picture. This could work, for example, by creating a collage of tracks (or albums) included in the playlist or by using unique pattern/artwork generators.

### Solution

When editing a playlist and adding an image (as described in the issue mentioned), Symphony should show the option to generate a picture, next to just choosing a file. There, an user could choose between various methods.

### Alternatives

An alternative would be to simply add instructions, so that users could compile their own artwork themselves.

### Additional Context

The most common variant of this (from an user perspective) seems to be a compilation of music artwork of the music first added to the playlist. Tangentially, GitHub has been generating a simple profile picture randomly, upon registration.

### Checklist

-   [x] I have read the [Contribution Guidelines](https://github.com/zyrouge/symphony/wiki/Contributions-Guidelines#issues).

**Comment by zyrouge** [comment]
A collage seems like a great idea.

**Comment by Currrupted** [comment]
> A collage seems like a great idea.

I agree! Sorry, I just saw that Symphony does in fact already use the song art of the first item of the playlist for its picture. I like this a lot! So this issue is mainly about a compilation/collage or other options an user could choose for generating the picture to a playlist.
